packages/frontend-shared/src/graphql/urqlClient.ts

```diff
@@ -103,7 +103,6 @@ declare global {
     __RUN_MODE_SPECS__: SpecFile[]
     __CYPRESS_TESTING_TYPE__: 'e2e' | 'component'
     __CYPRESS_BROWSER__: Partial<Browser> & {majorVersion: string | number}
-    __CYPRESS_TELEMETRY__?: {context: {traceparent: string}}
```

Comment:
removed duplicative typing, at least as far as i can tell we don't need it with the type defined in browsers.ts